frontend/src/v5/ui/routes/viewer/tickets/ticketsForm/properties/basicTicketImage/basicTicketImage.component.tsx

```diff
@@ -47,15 +47,17 @@ export const BasicTicketImage = ({
 	return (
 		<FormControl error={error} required={required}>
 			<Container className={className} error={error} disabled={disabled}>
-				<ActionsSide>
-					<Label>{label}</Label>
-					<ActionsList>{children}</ActionsList>
-				</ActionsSide>
-				<TicketImageDisplayer
-					imgSrc={value}
-					disabled={disabled || !isProjectAdmin}
-					onEmptyImageClick={uploadImage}
-				/>
+				<Label>{label}</Label>
+				<Content>
+					<TicketImageDisplayer
+						imgSrc={value}
+						disabled={disabled || !isProjectAdmin}
```

Comment:
This disabled value doesn't work. (I have viewer permissions)
![image](https://github.com/3drepo/3drepo.io/assets/62481250/fed884a5-e5b3-4df7-b6e2-c12390cda468)
The cursor doesn't change, but I can still click on it to open a file explorer window. Also should we remove the "Add image" text if the user doesn't have permissions?

You can just add `pointer-events: none;` to the styled component when disabled